### PR TITLE
Create few menu pages with curated posts and updated dropdowns.

### PR DESCRIPTION
**Pull request by Srikanth635**
Duplicate content templates are present in my repo, will be removed in further updates.

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-baseurl = "https://intel4coro.github.io/ViB-public/"
+baseurl = "https://srikanth635.github.io/ViB-public/"
```
Comment (yxzhan) [inline config.toml:1]: It is not necessary to have the whole domain name in the baseurl.
Please updated it as below to make the config works on both repos.
```
baseurl = "ViB-public/"
```

File: layouts/shortcodes/action_forms.html
Comment (yxzhan) [inline layouts/shortcodes/action_forms.html:1]: It seems like the select boxs are hardcoded in a  a fixed hierarchy,  Environment =>  Robot => Task.
It is OK to do so for now. But if there are new types of options, this widget will likely need to be rewritten.

**Review by yxzhan — CHANGES_REQUESTED** [review]
Please update `.github/workflows/hugo.yml` and `config.toml` before the updates can be merged.

File: layouts/shortcodes/action_forms.html
Comment (Srikanth635) [reply layouts/shortcodes/action_forms.html:1]: Yeah, for now this works. I'll rework on it.

**Pull request closed without merging**